test/util/util.go

```diff
@@ -95,3 +97,15 @@ func ParseCgroupFile(text string) map[string]string {
 	}
 	return cgroups
 }
+
+// TmpFileWithContent provide tmp file
+func TmpFileWithContent(content string) string {
+	tmpFile, err := ioutil.TempFile("", "envfile-test")
+	if err != nil {
+		log.Fatal(err)
```

Comment:
I'm afraid it is not necessary to log err. How about return err, and assert error in function TestRunWithEnvfile?